ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -91,7 +91,9 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
     public ngOnInit(): void {
         this.getActivity();
         this.initStepperState();
-        const submitSub = this.submitAttempted.subscribe(() => this.submitService.announceSubmit(null));
+        const submitSub = this.submitAttempted.subscribe((isSubmit: boolean) => {
+            this.submitService.announceSubmit(isSubmit);
```

Comment:
We  use it for emitting a submit event (`isSumbit = true`).
`isSumbit = false` is used when navigating Next/Prev

Reply:
should we do the same fix in atcp activity component?